crates/bevy_image/src/image.rs

```diff
@@ -761,6 +761,26 @@ impl Image {
         value
     }
 
+    /// Create a new zero-filled image with a given size, which can be rendered to.
+    /// This is primarily for use as a render target for a [`Camera`](`bevy_render::camera::Camera`).
+    /// See [`RenderTarget::Image`](`bevy_render::camera::RenderTarget::Image`).
```

Comment:
This fixes those pesky doc links :)

```suggestion
  /// Create a new zero-filled image with a given size, which can be rendered to. This is primarily
  /// for use as a render target for a [`Camera`]. See [`RenderTarget::Image`].
  ///
  /// [`Camera`]: https://docs.rs/bevy/latest/bevy/render/camera/struct.Camera.html
  /// [`RenderTarget::Image`]: https://docs.rs/bevy/latest/bevy/render/camera/enum.RenderTarget.html#variant.Image
```